src/wp-admin/includes/class-wp-theme-install-list-table.php

```diff
@@ -148,7 +148,19 @@ public function prepare_items() {
 		$api = themes_api( 'query_themes', $args );
 
 		if ( is_wp_error( $api ) ) {
-			wp_die( $api->get_error_message() . '</p> <p><a href="#" onclick="document.location.reload(); return false;">' . __( 'Try Again' ) . '</a>' );
+			wp_die( $api->get_error_message() . '</p> <p><a href="#" id="theme-try-again">' . __( 'Try Again' ) . '</a>' );
+
+			$js = <<<'JS'
```

Comment:
I assume this can't be in an external file?

Reply:
I couldn't find instances of this method being used, so I moved the handler to an inline script for simplicity.